noxfile.py

```diff
@@ -180,3 +181,38 @@ def make(session: nox.Session):
         *(args.make_args or ("clean", "coredocs")),
     ]
     session.run("make", "-C", "docs/docsite", *make_args, external=True)
+
+
+def _rm_tmpdir(tmpdir):
+    rmtree(tmpdir, ignore_errors=True)
+
+
+@nox.session
+def tag(session: nox.Session):
+    """
+    Determine the missing ansible-core releases,
+    create corresponding tags for each release in the ansible-documentation repo,
+    and then push them
+    """
+    install(session, req="tag")
+
+    DEFAULT_REPO = "https://github.com/ansible/ansible"
+    CHECK_LOCATION = Path.cwd().parents[0] / "ansible"
+
+    args = list(session.posargs)
```

Comment:
Might be an idea to check if git is installed. I think this could work:

```
if not shutil.which("git"):
    session.error("git is not installed or not found in PATH.")
```